ci/test/00_setup_env_native.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export CONTAINER_NAME=ci_native
+export DOCKER_NAME_TAG=ubuntu:20.04
+export PACKAGES="libqt5gui5 libqt5core5a qtbase5-dev libqt5dbus5 qttools5-dev qttools5-dev-tools libqt5charts5-dev libssl-dev libevent-dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-iostreams-dev libboost-program-options-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libqrencode-dev libzip-dev zlib1g zlib1g-dev libcurl4 libcurl4-openssl-dev"
+export RUN_UNIT_TESTS=false
```

Comment:
This is not exactly the same as our dependency list in .travis.yml. Also, shouldn't RUN_UNIT_TESTS be true here (and maybe false elsewhere?

Reply:
It was originally turned on, I turned it off after getting the `test/fs_tests.cpp` error in the x86_64 build.